### DISCUSSION TITLE
Prevent menu to close when clicking on an item

### DISCUSSION THREAD

**Discussion by verdie-g** [microsoft/fluentui-blazor#1995] [Q&A]
In the demos of [FluentMenu](https://www.fluentui-blazor.net/Menu), you can see that clicking on an item always close the menu. Is there a way to have a menu of checkboxes that:
- opens with a button click
- closes with a button click
- closes with a button outside of the menu
- does not close by clicking on a checkbox item

**Comment by dvoituron** [comment]
You have to use a [FluentPopover](https://www.fluentui-blazor.net/Popover) component. You have an example with checkboxes on this page: https://www.fluentui-blazor.net/Popover

**Reply by verdie-g** [reply]
Awesome thanks!
